### PR TITLE
Vpn suggestion updates

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Added new VPN text and buttons with links.

### CODE REVIEW

File: android/res/drawable/vpn_suggestion_button_background_selector.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+-->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@drawable/vpn_suggestion_button_background_selector_off" android:state_pressed="true" />
+    <item android:drawable="@drawable/vpn_suggestion_button_background_selector_off" android:state_focused="true" />
+    <item android:drawable="@drawable/vpn_suggestion_button_background_selector_on" />
+</selector>
```
Comment (aldenml) [inline android/res/drawable/vpn_suggestion_button_background_selector.xml]: missing `\n`

File: android/res/drawable/vpn_suggestion_button_background_selector.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), and
+ * Marcelina Knitter (@marcelinkaaa)
+ * Copyright (c) 2011, 2012, FrostWire(TM). All rights reserved.
```
Comment (aldenml) [inline android/res/drawable/vpn_suggestion_button_background_selector.xml]: wrong header

File: android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java [hunk 3]
```diff
@@ -77,40 +67,52 @@ protected void initComponents(Bundle savedInstanceState) {
...
-        OnGetVPNClickListener l = new OnGetVPNClickListener(isProtectedConnection, VPN);
-        headerIcon.setOnClickListener(l);
-        headerStatus.setOnClickListener(l);
-        getVPNButtonTop.setOnClickListener(l);
-        getVPNButtonBottom.setOnClickListener(l);
-        learnVPNButton.setOnClickListener(l);
-        vpnClientName.setText(VPN.companyNameStrResourceId);
+        learnVPNText.setPaintFlags(learnVPNText.getPaintFlags() |   Paint.UNDERLINE_TEXT_FLAG);
+
+        PiaVPN.setOnClickListener(new View.OnClickListener() {
```
Comment (aldenml) [inline android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java]: I'm surprised you are not getting nice hints about using lambda in the IDE

**Comment by aldenml** [comment]
Take care of the headers, dates and `<!--` at a new line.

File: android/res/drawable/vpn_suggestion_button_background_selector.xml (hunk 2 above)
Comment (marcelinkaaa) [reply android/res/drawable/vpn_suggestion_button_background_selector.xml]: Its this one, right? With just 2018 instead?
""

**Pull request merged**